testdata/_executor.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+error_log("executor.php called\n", 4);
+
 $fn = require $_SERVER['SCRIPT_FILENAME'];
 if ('1' !== ($_SERVER['FRANKENPHP_WORKER'] ?? null)) {
     $fn();
```

Comment:
Maybe something is going on `frankenphp_handle_request()`?

Reply:
My current theory, from what I can guess from the logs I added everywhere, is that it's an issue in our test suite: the main test thread is blocked for some reason.

Reply:
Hmmm. Maybe try downgrading go and see if it is a (recent) golang bug? I tried messing with GOMAXPROCS to see if maybe it was a thread starvation issue (because that is kinda what it looks like thanks to all your logging). But I suspect it that the <-done never happens in worker.go (line 152ish). I don't see `l.Debug("shutting down", zap.String("worker", fc.scriptFilename))` in the test output.

Reply:
It's unlikely a bug in Go because the main branch is green.

Reply:
The logs show that it's `frankenphp.Init()` that blocks. "FrankenPHP started 🐘" is never displayed. I'll add even more logs 😅

Reply:
ok... I know have the same issue with #909 locally, which doesn't include this patch. So... It's probably unrelated.